diff --git a/js/dropzone.integration.js b/js/dropzone.integration.js
index 30927d3..3409380 100644
--- a/js/dropzone.integration.js
+++ b/js/dropzone.integration.js
@@ -84,6 +84,10 @@
           uploadedFilesElement.attr('value', currentValue + response.result + ';');
         });

+        dropzoneInstance.on('error', function (file, message) {
+          dropzoneInstance.removeFile(file);
+        });
+
         // React on file removing.
         dropzoneInstance.on('removedfile', function (file) {
           var uploadedFilesElement = selector.siblings(':hidden');
